src/zjs_ashell.json

```diff
@@ -28,34 +29,30 @@
         "./deps/ihex/kk_ihex_read.c"
     ],
     "zephyr_conf": {
-        "arduino_101": [
+        "all": [
             "CONFIG_EARLY_CONSOLE=y",
             "CONFIG_CONSOLE_HANDLER=y",
             "CONFIG_CONSOLE_HANDLER_SHELL=y",
             "CONFIG_IHEX_UPLOADER_DEBUG=n",
             "CONFIG_REBOOT=y",
             "CONFIG_USB=y",
-            "CONFIG_USB_DW=y",
             "CONFIG_USB_DEVICE_STACK=y",
-            "CONFIG_USB_DW_DEBUG=n",
             "CONFIG_USB_DEBUG=n",
             "CONFIG_SERIAL=y",
             "CONFIG_UART_INTERRUPT_DRIVEN=y",
             "CONFIG_UART_LINE_CTRL=y",
             "CONFIG_USB_CDC_ACM_DEBUG=n",
             "CONFIG_DEVICE_POWER_MANAGEMENT=y",
-            "CONFIG_FILE_SYSTEM=y",
-            "CONFIG_FAT_FILESYSTEM_ELM=y",
-            "CONFIG_DISK_ACCESS_FLASH=y",
-            "CONFIG_FS_FAT_FLASH_DISK_W25QXXDV=y",
-            "CONFIG_FLASH=y",
```

Comment:
What are these changes about?

Reply:
I just removed individual fs configs and added dependency on ZJS_FS module directly, so no change in the functionality.